PrebidMobile/PrebidMobileCore/src/main/java/org/prebid/mobile/core/TargetingParams.java

```diff
@@ -111,6 +113,53 @@ static void fetchLocationUpdates(Context appContext) {
     //endregion
 
     //region Public APIs
+    static final String PREBID_CONSENT_STRING_KEY = "Prebid_GDPR_consent_strings";
+    static final String IABConsent_ConsentString = "IABConsent_ConsentString";
+
+    public static void setGDPRConsentString(Context context, String string) {
+        if (!TextUtils.isEmpty(string) && context != null) {
+            SharedPreferences pref = PreferenceManager.getDefaultSharedPreferences(context);
+            SharedPreferences.Editor editor = pref.edit();
+            editor.putString(PREBID_CONSENT_STRING_KEY, string);
+            editor.apply();
+        }
+    }
+
+    public static String getGDPRConsentString(Context context) {
+        if (context != null) {
+            SharedPreferences pref = PreferenceManager.getDefaultSharedPreferences(context);
+            if (pref.contains(PREBID_CONSENT_STRING_KEY)) {
+                return pref.getString(PREBID_CONSENT_STRING_KEY, "");
+            } else if (pref.contains(IABConsent_ConsentString)) {
+                return pref.getString(IABConsent_ConsentString, "");
+            }
+        }
+        return null;
+    }
+
+    static final String PREBID_GDPR_KEY = "Prebid_GDPR";
+    static final String IABConsent_SubjectToGDPR = "IABConsent_SubjectToGDPR";
+
+    public static void setSubjectToGDPR(Context context, boolean consent) {
+        if (context != null) {
+            SharedPreferences pref = PreferenceManager.getDefaultSharedPreferences(context);
+            SharedPreferences.Editor editor = pref.edit();
+            editor.putBoolean(PREBID_GDPR_KEY, consent);
+            editor.apply();
+        }
+    }
+
+    public static Boolean isSubjectToGDPR(Context context) {
+        if (context != null) {
+            SharedPreferences pref = PreferenceManager.getDefaultSharedPreferences(context);
+            if (pref.contains(PREBID_GDPR_KEY)) {
+                return pref.getBoolean(PREBID_GDPR_KEY, false);
+            } else if (pref.contains(IABConsent_SubjectToGDPR)) {
```

Comment:
The IAB value is not a boolean its a string and its values can be 
1 - Subject to GDPR
0 - Not Subject to GDPR
nil - unknown or not set

Reply:
Yes if it doesn't contain the key then return null at the end.

Reply:
pref.getBoolean(IABConsent_SubjectToGDPR, false);

I was concerned about this since the getBoolean is not gonna work.  The value present inside the shared preference is not a Boolean but a String.